docker-compose.yml

```diff
@@ -6,8 +6,11 @@ services:
     container_name: fw-exports-develop
     environment:
       PORT: 3000
-      AUTH_URL: https://api.resourcewatch.org
+      AUTH_URL: https://staging-api.resourcewatch.org
       FORMS_API_URL: https://api.resourcewatch.org/v1
+      AREAS_API_URL: "https://dev-fw-api.globalforestwatch.org/v3/forest-watcher"
+      ALERTS_API_URL: "https://staging-data-api.globalforestwatch.org/"
```

Comment:
Need to remove the ending `/` to avoid `//` issues with API calls
`"https://staging-data-api.globalforestwatch.org"`